docs/source/sysadmin_guide/pbft_adding_removing_node.rst

```diff
@@ -0,0 +1,158 @@
+   setting (see :doc:`configuring_sawtooth/path_configuration_file`).
+
+#. On an existing member node, update ``sawtooth.consensus.pbft.members`` to
+   include the public validator key of the new node.
+
+   Run the following steps on a node that has permission to change on-chain
+   settings; that is, a node whose validator key is listed in
+   ``sawtooth.identity.allowed_keys``. Usually, the node that created the
+   genesis block is on this list. For more information, see
+   :ref:`config-onchain-txn-perm-label`.
+
+   a. List the current PBFT member nodes:
+
+      .. code-block:: console
+
+         $ sawtooth setting list --filter sawtooth.consensus.pbft.members
+
+   #. Submit a transaction that specifies the new list of all PBFT member nodes
+      (the previous list plus the new node's key).
+
+      .. Important::
+
+         BE VERY CAREFUL! Make sure to specify the full list of keys.
+         Double-check each key before you run this command, because a typo could
+         stall the network.
+
+      .. code-block:: console
+
+         $ sawset proposal create \
+           --key /etc/sawtooth/keys/validator.priv \
+           sawtooth.consensus.pbft.members=[previous-list, NEW-KEY]
+
+      If there are no errors, the block containing this setting change will be
+      committed quickly.
+
+#. When all nodes have detected the change and updated their local copy of the
+   member list, the new member node begins to participate in the PBFT network.
+
+.. _removing-a-pbft-node-label:
+
+Removing a PBFT Node
+====================
+
+To remove an existing node from a PBFT network, you will delete the node's
+validator key from  the ``sawtooth.consensus.pbft.members`` setting, then shut
+down the removed node. You can delete several nodes at the same time.
+
+.. Note::
+
+   PBFT consensus requires a network with at least four nodes. A network with
+   fewer than four nodes will fail.
+
+1. Update ``sawtooth.consensus.pbft.members`` to no longer include the
+   validator public key of the node you want to remove.
+
+   a. List the current PBFT member nodes:
+
+      .. code-block:: console
+
+         $ sawtooth setting list --filter sawtooth.consensus.pbft.members
+
+   #. Submit a transaction that specifies the new list of all PBFT member nodes
+      (the previous list, minus the key of the node or nodes to be removed).
+
+      .. Important::
+
+         BE VERY CAREFUL! Make sure to specify the correct list of keys.
+         Double-check each key before you run this command, because a typo could
+         stall the network.
+
+      .. code-block:: console
+
+         $ sawset proposal create \
+           --key /etc/sawtooth/keys/validator.priv \
+           sawtooth.consensus.pbft.members=[UPDATED-LIST]
+
+      If there are no errors, the block containing this setting change will be
+      committed quickly.
+
+#. Make sure that change has been committed. You can check the setting from
+   another node.
```

Comment:
You can check the setting from any node, even one that's been removed from the list (because they still commit the block with the settings change). So I'd just say you can check the setting from any node.